templates/step/edit.html.ep

```diff
@@ -83,18 +83,20 @@
                     % for my $tag (@$tags) {
                         <label>
                             <input type="checkbox" name="tags" id="tag_<%= $tag %>" class="tag_checkbox"
-                                   value="<%= $tag %>"><%= $tag %>
+                                   value="<%= $tag %>" <%= 'readonly="readonly"' unless (is_operator) %> ><%= $tag %>
```

Comment:
for checkboxes you need to use `disabled` too. Otherwise they are greyed out but still state can be changed.